### PR TITLE
Beaconing - Update whitelist in transform

### PR DESCRIPTION
**Pull request by susan-shu-c**
## Proposed commit message

Update exceptionlist in the beaconing detection package to reduce false positives. Adds two processes.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] 

## How to test this PR locally

```
elastic-package stack up -d -v --version 8.11.1
```

## Related issues

- https://github.com/elastic/detection-rules/pull/3497

## Screenshots

Screenshot 2024-03-12 at 1 47 32 PM

### CODE REVIEW

File: packages/beaconing/docs/README.md [hunk 1]
```diff
@@ -48,8 +48,18 @@ The transform, which runs every hour, also filters out common, known application
 
 The values highlighted above are typical of beaconing behavior and can help with your investigation.
 
+## Exceptionlist customizations
+
+There are two places where you can customize which processes are on an exceptionlist for this detection. The first is in the transforms: these contain processes which are common, and thus it is beneficial for performance and data processing to keep some processes here. You can read more on how to customize the transforms in the next section.
```
Comment (sodhikirti07) [inline packages/beaconing/docs/README.md:53]:
```suggestion
There are two places where you can customize which processes are on an exceptionlist for this detection. The first is in the transforms: these contain processes which are common, and thus it is beneficial for the transform's performance and data processing to keep some processes here. You can read more on how to customize the transforms in the next section.
```

File: packages/beaconing/docs/README.md [hunk 2]
```diff
@@ -48,8 +48,18 @@ The transform, which runs every hour, also filters out common, known application
...
+
 ## Further customizations
-Advanced users can also tune the scripted metric aggregation's parameters, such as jitter percentage or time window. To overwrite the default parameters: delete the transform, change the parameters, and restart the transform. The configurable parameters are:
+
+Advanced users can also tune the scripted metric aggregation's parameters, such as jitter percentage or time window. Navigate to Transforms on your Elastic cluster and find the transform installed by this package (search `beaconing`). You can browse the source of the transform there by clicking on the `.json` tab; the source code of the latest version can also be viewed [here](https://github.com/elastic/integrations/blob/main/packages/beaconing/elasticsearch/transform/pivot_transform/transform.yml#L364).
+
+To overwrite the default parameters: delete the transform, change the parameters, and restart the transform. The configurable parameters are:
+
```
Comment (sodhikirti07) [inline packages/beaconing/docs/README.md:62]: Do you mean `close` the transform instead of `delete`?

Comment (susan-shu-c) [reply packages/beaconing/docs/README.md:62]: I think actually the steps are stop the transform then delete: UI won't let you delete unless it's stopped - I can add this
 (this was in the original doc, not sure why it's in the git diff)

**Review by sodhikirti07 — APPROVED** [review]
Changes looks good to me!

**Comment by susan-shu-c** [comment]
Will merge after this one is merged, so that the documentation in this one won't be inaccurate
https://github.com/elastic/detection-rules/pull/3497

**Pull request merged**